Resources/Prototypes/Atmospherics/reactions.yml

```
@@ -91,6 +91,79 @@
+  - 0     # miasma
+  - 0     # n2o
+  - 0     # frezon
+  - 5     # bz
+  effects:
+  - !type:NitriumProductionReaction {}
+
+- type: gasReaction
+  id: NitriumDecomposition
+  priority: 2
+  minimumRequirements:
+  - 0.01  # oxygen
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)